### PR TITLE
Add de.marlam.gencolormap

### PR DESCRIPTION
**Pull request by marlam**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: de.marlam.gencolormap.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+sdk: org.kde.Sdk
+command: gencolormap-gui
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # Save color maps to xdg-pictures
+  - --filesystem=xdg-pictures:rw
```
Comment (hfiguiere) [inline de.marlam.gencolormap.yml:13]:
```suggestion
  - --filesystem=xdg-pictures
```

**Comment by hfiguiere** [comment]
bot, build de.marlam.gencolormap

**Comment by hfiguiere** [comment]
```
builddir/files/share/appdata/de.marlam.gencolormap.appdata.xml: FAILED:
• style-invalid         : <caption> cannot end in '.' [The user interface.]
Validation of files failed
```

File: de.marlam.gencolormap.yml (hunk 1 above)
Comment (marlam) [reply de.marlam.gencolormap.yml]: Done, thanks.

**Comment by barthalion** [comment]
bot, build de.marlam.gencolormap

**Pull request closed without merging**